### PR TITLE
show deploy groups for each stage on project overview page

### PR DESCRIPTION
**Pull request by jonmoter**
also rename headers to be more useful

/cc @zendesk/runway 

### Risks
 - None

### CODE REVIEW

**Comment by jonmoter** [comment]
example-project

File: app/views/projects/_stage.html.erb [hunk 1]
```diff
@@ -4,12 +4,17 @@
       <%= link_to stage.name, project_stage_path(project, stage) %>
       <%= stage_lock_icon stage %>
     </td>
-    <% if stage.last_successful_deploy %>
-      <td><%= stage.last_successful_deploy.user.name %></td>
-      <td><%= link_to stage.last_successful_deploy.short_reference, project_deploy_path(project, stage.last_successful_deploy), title: "#{stage.last_successful_deploy.reference} at #{stage.last_successful_deploy.updated_at.to_s(:db)}" %></td>
+    <% if ENV['DEPLOY_GROUP_FEATURE'] %>
+      <td><%= stage.deploy_groups.map(&:name).join(', ') %></td>
+    <% end %>
+    <% if (deploy = stage.last_successful_deploy) %>
```
Comment (grosser) [inline app/views/projects/_stage.html.erb]: the parentheses are unnecessary

**Comment by grosser** [comment]
:+1:

File: app/views/projects/_stage.html.erb (hunk 1 above)
Comment (jonmoter) [inline app/views/projects/_stage.html.erb]: Yeah, not technically necessary. At some point I picked up the habit of adding parentheses when doing an assignment inside of a conditional statement, to have it stand out a little bit.

**Pull request merged**